packages/webapp/src/components/LocationPicker/SingleLocationPicker/drawLocations.js

```diff
@@ -20,37 +22,42 @@ export const SELECTED_POLYGON_OPACITY = 1.0;
 export const DEFAULT_POLYGON_OPACITY = 0.5;
 export const HOVER_POLYGON_OPACITY = 0.8;
 
-export const drawCropLocation = (map, maps, mapBounds, location) => {
-  if (isNoFillArea(location.type)) return drawNoFillArea(map, maps, mapBounds, location);
-  if (isLine(location.type)) return drawLine(map, maps, mapBounds, location);
-  if (isArea(location.type)) return drawArea(map, maps, mapBounds, location);
-  if (isPoint(location.type)) return drawPoint(map, maps, mapBounds, location);
+export const drawCropLocation = (map, maps, mapBounds, location, disableHover = false) => {
```

Comment:
Should this be renamed to `drawLocation` instead since it's not strictly related to crops?